src/main/java/com/breskul/bibernate/persistence/EntityMapper.java

```diff
@@ -0,0 +1,48 @@
+package com.breskul.bibernate.persistence;
+
+
+import com.breskul.bibernate.exception.EntityQueryException;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.List;
+
+public class EntityMapper<T> {
+
+    private final Class<T> entityCls;
+    private final List<Field> columnFields;
+
+    public EntityMapper(Class<T> entityCls, List<Field> columnFields) {
+        this.entityCls = entityCls;
+        this.columnFields = columnFields;
+    }
+
+    public T mapResult(ResultSet resultSet) {
+        try {
+            T t = entityCls.getConstructor().newInstance();
+            for (int i = 0; i < columnFields.size(); i++) {
+                Field field = columnFields.get(i);
+                field.setAccessible(true);
+                field.set(t, resultSet.getObject(i + 1));
```

Comment:
Add todo or implement that we need to get here field names from `@Column` annotation (or get field name) and get data from resultSet By field name